tests/unit/amo/reducers/testFeatured.js

```diff
@@ -20,7 +20,9 @@ describe('featured reducer', () => {
   describe('FEATURED_GET', () => {
     it('sets the initialState', () => {
       const { addonType, loading, results } = featured(
-        initialState, actions.getFeatured({ addonType: 'theme' }));
+        initialState, actions.getFeatured({
+          addonType: 'theme', errorHandlerId: 'some-error-handler',
```

Comment:
This should use a constant rather than a hard-coded string, as I can't tell whether this should be an `ADDON_TYPE_COMPLETE_THEME` or `ADDON_TYPE_PERSONA` from the context.